backend/nfconfig/config.go

```diff
@@ -5,12 +5,136 @@
 
 package nfconfig
 
-type AccessAndMobilityConfig struct{}
+import (
+	"sort"
+	"strconv"
 
-type PlmnConfig struct{}
+	"github.com/omec-project/openapi/nfConfigApi"
+	"github.com/omec-project/webconsole/backend/logger"
+	"github.com/omec-project/webconsole/configmodels"
+)
 
-type PlmnSnssaiConfig struct{}
+type inMemoryConfig struct {
+	plmn              []nfConfigApi.PlmnId
+	plmnSnssai        []nfConfigApi.PlmnSnssai
+	accessAndMobility []nfConfigApi.AccessAndMobility
+	sessionManagement []nfConfigApi.SessionManagement
+	policyControl     []nfConfigApi.PolicyControl
+}
 
-type SessionManagementConfig struct{}
+func (c *inMemoryConfig) syncPlmn(slices []configmodels.Slice) {
+	plmnSet := make(map[nfConfigApi.PlmnId]bool)
+	newPlmnConfig := []nfConfigApi.PlmnId{}
+	for _, s := range slices {
+		plmn := *nfConfigApi.NewPlmnId(s.SiteInfo.Plmn.Mcc, s.SiteInfo.Plmn.Mnc)
+		if !plmnSet[plmn] {
+			plmnSet[plmn] = true
+			newPlmnConfig = append(newPlmnConfig, plmn)
+		}
+	}
 
-type PolicyControlConfig struct{}
+	sort.Slice(newPlmnConfig, func(i, j int) bool {
+		if newPlmnConfig[i].GetMcc() != newPlmnConfig[j].GetMcc() {
+			return newPlmnConfig[i].GetMcc() < newPlmnConfig[j].GetMcc()
+		}
+		return newPlmnConfig[i].GetMnc() < newPlmnConfig[j].GetMnc()
+	})
+
+	c.plmn = newPlmnConfig
+	logger.NfConfigLog.Debugln("Updated PLMN in-memory configuration. New configuration: ", c.plmn)
```

Comment:
There is no need for an empty space between `configuration:` and `"`
```suggestion
	logger.NfConfigLog.Debugln("Updated PLMN in-memory configuration. New configuration:", c.plmn)
```